crates/ibc/src/core/ics02_client/handler/update_client.rs

```diff
@@ -753,4 +866,66 @@ mod tests {
         assert!(res.is_ok());
         ensure_misbehaviour(&ctx_a, &client_id, &tm_client_type());
     }
+
+    #[test]
+    fn test_expired_client() {
+        let remote_chain_id = ChainId::new("mockgaiaB", 1).unwrap();
+        let host_chain_id = ChainId::new("mockgaiaA", 1).unwrap();
```

Comment:
To be consistent with other places, could we keep using [our naming convention](https://github.com/cosmos/ibc-rs/blob/dc8e4161dd5e00a2031cf3feb58dad1ed7e26bdb/crates/ibc/src/core/mod.rs#L3-L23) instead of remote and host?